src/app/api/v1/realtime/collections.py

```diff
@@ -215,6 +219,12 @@ async def material_counts_tree(
     status_code=HTTP_200_OK,
     responses={HTTP_404_NOT_FOUND: {"description": "Collection not found"}},
     tags=["Statistics"],
+    description="""Returns the average ratio of non-empty properties.
```

Comment:
I would phrase it as
> Returns the average ratio of non-empty properties for the collection.
We could also rename the `noreref_id` as `collection_id`?

Reply:
Good point. I will change this.

Reply:
Done